src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -39,13 +39,13 @@ class WebDebugToolbarListener implements EventSubscriberInterface
     public const DISABLED = 1;
     public const ENABLED = 2;
 
-    protected $twig;
-    protected $urlGenerator;
-    protected $interceptRedirects;
-    protected $mode;
-    protected $excludedAjaxPaths;
-    private $cspHandler;
-    private $dumpDataCollector;
+    protected Environment $twig;
+    protected ?UrlGeneratorInterface $urlGenerator;
+    protected bool $interceptRedirects;
```

Comment:
Given that the class is tagged as `@final`, shouldn't it become actually final in 6.0 and those properties become private ?